docs/docs/reference/other-new-features/safe-initialization.md

```diff
@@ -29,11 +29,11 @@ The checker will report:
 
 ``` scala
 -- Warning: tests/init/neg/AbstractFile.scala:7:4 ------------------------------
-7 |	val localFile: String = url.hashCode + ".tmp"  // error
-  |	    ^
-  |    Access non-initialized field value localFile. Calling trace:
-  |     -> val extension: String = name.substring(4)	[ AbstractFile.scala:3 ]
-  |      -> def name: String = localFile            	[ AbstractFile.scala:8 ]
+7 |	  val localFile: String = s"${url.##}.tmp"    // error: usage of `localFile` before it's initialized
```

Comment:
Here is compiler output. Please unfold the example code above, and you'll see that code doesn't match this line.

After fixing this, I also modify the source of this example, _tests/init/neg/AbstractFile.scala_, and try to match them.